src/manager.spec.js

```diff
@@ -1111,136 +1118,53 @@ describe('AdManager', function() {
+    beforeEach(function() {
+	   var headertag = window.headertag = {};
+      sandbox = sinon.sandbox.create();
+      window.apstag = {
+        fetchBids: function () {
+			this.callArg(callback)
+		}
       };
 
-      domElement = document.createElement('div');
-      document.body.appendChild(domElement);
-      adManager.amazonAdRefresh(domElement);
-    });
-
-    it('calls setTargetingForGPTAsync', function () {
-      expect(adManager.amznads.setTargetingForGPTAsync.calledOnce).to.be.true;
-    });
-
-    it('calls refreshAds', function () {
-      expect(adManager.refreshAds.calledWith(domElement)).to.be.true;
-    });
-
-    it('clears targeting', function () {
-      var expected = adManager.googletag.pubads().clearTargeting.calledOnce;
-      expect(expected).to.be.true;
-    });
-  });
-
-  describe('#doGetAmazonAdsCallback', function () {
-    var params;
-    beforeEach(function () {
-      adManager.amznads = {
-        getAdsCallback: sinon.stub()
+      headertag.cmd = {
+        push: function() {
+         window.apstag.setDisplayBids()
+        }
       };
-      params = { id: 1, callback: function () {}, timeout: 1e4 };
-    });
 
-    it('sets lastGetAdsCallback', function () {
-      expect(adManager.amznads.lastGetAdsCallback).to.be.undefined;
-      adManager.doGetAmazonAdsCallback(params);
-      var expected = typeof adManager.amznads.lastGetAdsCallback === 'number';
-      expect(expected).to.be.true;
+      sandbox.stub(window.headertag.cmd, 'push')
+      sandbox.stub(window.apstag, 'fetchBids');
     });
 
-    it('calls amazons getAdsCallback', function () {
-      adManager.doGetAmazonAdsCallback(params);
-      expect(adManager.amznads.getAdsCallback.calledOnce).to.be.true;
+    afterEach(function() {
+      sandbox.restore();
     });
-  });
-
-  describe('#amazonAdRefreshThrottled', function () {
-    var params, clock;
 
-    beforeEach(function () {
-      adManager.amznads = { getAdsCallback: sinon.stub() };
-      params = { id: 1, callback: sinon.stub(), timeout: 1e4 };
+    it('call to apstag functions if apstag is defined', function () {
+      adManager.fetchAmazonBids();
+      expect(window.apstag.fetchBids.called).to.be.true;
     });
 
-    context('calls doGetAmazonAdsCallback if lastGetAdsCallback', function () {
-      it('is undefined', function () {
-        adManager.amazonAdRefreshThrottled(params);
-        expect(adManager.amznads.getAdsCallback.calledOnce).to.be.true;
-      });
-
-      it('is more than 10 seconds old', function () {
-        clock = sinon.useFakeTimers(2e4);
-        adManager.amznads.lastGetAdsCallback = 0;
-        adManager.amazonAdRefreshThrottled(params);
-        expect(adManager.amznads.getAdsCallback.calledOnce).to.be.true;
-        clock.restore();
-      });
+    it('fetches bids from apstag api', function (done) {
+      adManager.fetchAmazonBids();
+      callback();
+      expect(window.headertag.cmd.push.calledOnce).to.be.true;
+      done();
     });
```

Comment:
@spra85 The formatting is a bit hard to read.  Here's a link to the file itself: https://github.com/theonion/bulbs-public-ads-manager/blob/5045f40b0e94ab352ec6b0970bfe47bfce1a06b8/src/manager.spec.js#L1126

I was discussing this test @ashtonG who commented that this test doesn't really test anything useful since all of the code is external. After that discussion i kind of think we should remove this test. If we think its necessary we could test the api calls which look something like this:
`[{ amznbid:"testBid",
amzniid:"testImpression-6943214141509574215826",
amznp:"testP",
amznsz:"728x90",
size:"728x90",
slotID:"dfp-ad-2" }]`

That said, we don't actually do anything the bid response since we use amazon new implementation where the key vals are appended to the dfp ad calls automatically.